### ISSUE TITLE
Help - GTA5 on steam unlocking after reset?

### ISSUE THREAD

**Issue by azzaroo** [gibbed/SteamAchievementManager#184]
SAM works perfectly fine, I know this is an issue with the rockstar launcher with the reunlocking of achievements but has anyone came across a way? I reset my steam achievements for GTA 5 and it worked but as I started the game again they just reunlock, I deleted my save files from C: drive but i believe it has something to do with the rockstar launcher. Anyone know of a way to stop this happening? @gibbed

**Comment by treymorris** [comment]
Turn off cloud sync. Deleting them locally won't matter since they're just going to be downloaded from the cloud when you launch it. You should backup your save(s) first.

You can view your Steam cloud saves here as well: https://store.steampowered.com/account/remotestorage

**Comment by WollyBog45** [comment]
> Turn off cloud sync. Deleting them locally won't matter since they're just going to be downloaded from the cloud when you launch it. You should backup your save(s) first.
> 
> You can view your Steam cloud saves here as well: https://store.steampowered.com/account/remotestorage

**Comment by WollyBog45** [comment]
Could you go into more detail about this?

**Comment by WollyBog45** [comment]
I know this is kind of late but I just wanted to say that it doesn't work. The reason that it worked for Baldur's Gate and not for GTA is because GTA's publisher (Rockstar Games) keeps there accomplishments stored in their database. And Baldur's Gate publisher (Larian Studios) does not (they keep it in steam's). Which means if you tried to remove GTA accomplishments with steam even if you turn off auto updates, turn off steam cloud sync, etc. It still wont work because the accomplishments are stored in Rockstar's database not steam's. I still hope you could find a way.

**Comment by WollyBog45** [comment]
Note: You should probably try it for your self in case I'm doing something wrong.

**Comment by ConnorGithub666** [comment]
made an account just to reply to this, I am running into the exact same issue.

Played GTAV story on PC originally with Epic Games, now years later I got it for cheap on Steam, tried to reset auto achievements given with SAM and it just readds them again, because of Rockstar keeping track of your achievements. It's a shame.

**Comment by syntax-tm** [comment]
Yeah if it's a cross platform thing then it's just going to load them from the server every time.

I'm not familiar with GTA but if you can unlink your Steam amount that might work. Whenever you reconnect it it's just going to use your Rockstar account (or whatever it is).

If you just want the ability to play the game and unlock stuff again one option would be to create a new Steam account and create a Steam Family to share your library. Not sure if that's enabled for that game or not but most allow it.

**Comment by syntax-tm** [comment]
Nevermind, family sharing is disabled for GTA V.

**Comment by teolabx** [comment]
I found an option to disable the automatic opening of achievements from the rockstar account. But achievements will no longer open automatically on steam, you will have to open each manually via SAM. But it's better than opening all at once at the same time through syncing with rockstar. Using the example of RDR2 . In the directory "C:\Users\USERNAME\OneDrive\Documents\Rockstar Games\Social Club\Profiles\********\ Titles\Red Dead Redemption 2" open the file "achievements.dat" using notepad. You need to delete everything. Click save. It is mandatory to set the "read-only" attributes to the file. That's it
